src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,232 +32,19 @@ const NavBar: React.FC<NavBarPropsType> = ({
-  const showRadioOverlay = useMemo(() => {
-    if (!radioFirstPlayStateLoaded.current) {
-      const radioFirstPlayStorageKey = "radioFirstPlay";
-      const radioFirstPlayState = localStorage.getItem(
-        radioFirstPlayStorageKey
-      );
-      if (!radioFirstPlayState) {
-        localStorage.setItem(radioFirstPlayStorageKey, JSON.stringify(true));
-        return true;
-      }
-      radioFirstPlayStateLoaded.current = true;
-    }
-    return false;
-  }, [radioFirstPlayStateLoaded]);
-
-  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
-    setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
-  const hideEventSchedule = useCallback(() => {
-    setEventScheduleVisible(false);
-  }, []);
-
-  const parentVenueId = venue?.parentId ?? "";
-  const backToParentVenue = useCallback(() => {
-    window.location.href = venueInsideUrl(parentVenueId);
-  }, [parentVenueId]);
-
-  const navigateToHomepage = useCallback(() => {
-    const venueLink =
-      redirectionUrl ?? venueId ? venueInsideUrl(venueId ?? "") : "/";
-
-    window.location.href = venueLink;
-  }, [redirectionUrl, venueId]);
-
-  const handleRadioEnable = useCallback(() => setIsRadioPlaying(true), []);
-
-  const [showRadioPopover, setShowRadioPopover] = useState(false);
-
-  const toggleShowRadioPopover = useCallback(
-    () => setShowRadioPopover((prevState) => !prevState),
-    []
-  );
-
-  if (!venueId || !venue) return null;
-
-  // TODO: ideally this would find the top most parent of parents and use those details
-  const navbarTitle = parentVenue?.name ?? venue.name;
-
-  const profileImage = profile?.pictureUrl || DEFAULT_PROFILE_IMAGE;
-
-  const radioStation = !!hasRadioStations && radioStations![0];
-
-  const showNormalRadio = (venue?.showRadio && !isSoundCloud) ?? false;
-  const showSoundCloudRadio = (venue?.showRadio && isSoundCloud) ?? false;
-
   return (
     <>
       <header>
-        <div className={`navbar navbar_playa ${!isOnPlaya && "nonplaya"}`}>
+        <div className={`navbar navbar_playa nonplaya`}>
           <div className="navbar-container">
-            <div className="nav-logos">
-              <div className="nav-sparkle-logo">
-                <div />
-              </div>
-              <div
-                className="nav-sparkle-logo_small"
-                onClick={navigateToHomepage}
-              >
-                <div />
-              </div>
-              <div
-                className={`nav-party-logo ${
-                  isEventScheduleVisible && "clicked"
-                }`}
-                onClick={toggleEventSchedule}
-              >
-                {navbarTitle} Schedule
-              </div>
-              <VenuePartygoers />
-            </div>
-
-            {!user && <NavBarLogin />}
-
-            {user && (
-              <div className="navbar-links">
-                <NavSearchBar />
```

Comment:
[note] Moved to `src/components/organisms/WithNavigationBar/WithNavigationBar.tsx`